src/Generator/PropertyGenerator.php

```diff
@@ -304,4 +329,17 @@ public function omitDefaultValue(bool $omit = true)
 
         return $this;
     }
+
+    public function getType(): ?string
+    {
+        return $this->type;
+    }
+
+    public function setType(?string $type): void
+    {
+        if (class_exists($type) && ! str_starts_with($type, '\\')) {
```

Comment:
`str_starts_with` not exists yet in php 7.4

Reply:
Thanks for quickly reviewing the code. 

symfony/polyfill-php80 package was included because of vimeo/psalm, so I presume it should be fine?

https://github.com/laminas/laminas-code/blob/c99ef8e5629c33bfaa3a8f1df773e916af564cd6/composer.json#L21

Reply:
I've updated the code to be compatible with PHP 7.4